### PR TITLE
QPPCT-549: remove multitin stuff

### PR DESCRIPTION
**Pull request by clydet**
### Information
- QPPCT-549

### Changes proposed in this PR:
- remove multi tin encoding / validation / encoding

### Checklist 
- [x] All JUnit tests pass (`mvn clean verify`).
- [ ] New unit tests written to cover new functionality.
- [ ] Added and updated JavaDocs for non-test classes and methods.
- [ ] No local design debt. Do you feel that something is "ugly" after your changes?
- [ ] Updated documentation (`README.md`, etc.) depending if the changes require it.

### CODE REVIEW

File: converter/src/main/java/gov/cms/qpp/conversion/encode/ClinicalDocumentEncoder.java [hunk 1]
```diff
@@ -91,7 +90,7 @@ private JsonWrapper encodeMeasurementSets(Map<TemplateId, Node> childMapByTempla
 		JsonOutputEncoder sectionEncoder;
 
 		for (Node child : childMapByTemplateId.values()) {
-			if (TemplateId.NPI_TIN_ID != child.getType()) {
+//			if (TemplateId.NPI_TIN_ID != child.getType()) {
```
Comment (sfradkin) [inline converter/src/main/java/gov/cms/qpp/conversion/encode/ClinicalDocumentEncoder.java]: remove commented code

File: converter/src/main/java/gov/cms/qpp/conversion/encode/ClinicalDocumentEncoder.java [hunk 2]
```diff
@@ -101,7 +100,7 @@ private JsonWrapper encodeMeasurementSets(Map<TemplateId, Node> childMapByTempla
 					String message = "No encoder for decoder : " + child.getType();
 					throw new EncodeException(message, exc);
 				}
-			}
+//			}
```
Comment (sfradkin) [inline converter/src/main/java/gov/cms/qpp/conversion/encode/ClinicalDocumentEncoder.java]: remove commented code

Comment (FlexionSonarQube) [inline converter/src/main/java/gov/cms/qpp/conversion/encode/ClinicalDocumentEncoder.java]: MAJOR This block of commented-out lines of code should be removed. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

File: converter/src/main/java/gov/cms/qpp/conversion/encode/ClinicalDocumentEncoder.java (hunk 1 above)
Comment (FlexionSonarQube) [inline converter/src/main/java/gov/cms/qpp/conversion/encode/ClinicalDocumentEncoder.java]: MAJOR This block of commented-out lines of code should be removed. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

**Comment by FlexionSonarQube** [comment]
SonarQube analysis reported 2 issues
* ![MAJOR][MAJOR] 2 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: converter/src/main/java/gov/cms/qpp/conversion/encode/ClinicalDocumentEncoder.java (hunk 1 above)
Comment (saquino0827) [inline converter/src/main/java/gov/cms/qpp/conversion/encode/ClinicalDocumentEncoder.java]: Delete this comment

**Comment by clydet** [comment]
@sfradkin regarding tin / npi  validations, @halprin pointed out a change set for QPPCT-195 that removed those as duplicates of submission api validations.

**Comment by halprin** [comment]
@sfradkin Here is the commit as part of QPPCT-195 that @clydet is talking about.  https://github.com/CMSgov/qpp-conversion-tool/commit/dc87cd32ec71bd73358870e014f778872436e7cf
I do agree though that we should have a test that verifies we only have one NPI given a multi-NPI CPC+ file.

**Comment by clydet** [comment]
ah, agreed

File: converter/src/main/java/gov/cms/qpp/conversion/decode/ClinicalDocumentDecoder.java [hunk 3]
```diff
@@ -152,7 +154,7 @@ private void processComponentElement(Element element, Node thisNode) {
 			pairs[1] = ENTITY_GROUP;
 		} else if (CPCPLUS.equalsIgnoreCase(name)) {
 			pairs[0] = CPCPLUS_PROGRAM_NAME;
-			pairs[1] = ENTITY_GROUP;
+			pairs[1] = ENTITY_INDIVIDUAL;
```
Comment (sfradkin) [inline converter/src/main/java/gov/cms/qpp/conversion/decode/ClinicalDocumentDecoder.java]: @clydet This somehow got changed back to individual here for CPCPLUS...

**Pull request merged**